script.js

```diff
@@ -0,0 +1,34 @@
+const menuBtn = document.querySelector('.hamburger');
+const mobileMenu = document.querySelector('.mobile-menu');
+const clsmodal = document.querySelector('#cls-modal');
+
+menuBtn.addEventListener("click", () => {
+  menuBtn.classList.toggle("active");
+  mobileMenu.classList.toggle("active");
+})
+
+clsmodal.addEventListener("click", () => {
+  mobileMenu.classList.remove("active")
+  menuBtn.classList.remove("active");
+});
```

Comment:
**[REQUIRED CHANGES]**

- Please, kindly ensure your design matches the Figma design as much as you can below:

- [ ] Use the same size for the cancel button.
- [ ] Rename the `Recent works` to `portfolio`

![seee](https://user-images.githubusercontent.com/59397159/174702957-6df40c68-831c-4373-98e5-d0695f3721f3.png)